electron/main.ts

```diff
@@ -788,6 +791,18 @@ ipcMain.handle('install', async (event, params) => {
     return
   }
 
+  const epicOffline = await isEpicOffline()
+  if (epicOffline) {
+    dialog.showErrorBox(
+      i18next.t('box.warning.title', 'Warning'),
+      i18next.t(
+        'box.warning.epic.install',
+        'Epic Servers are having major outage right now, the game cannot be installed!'
+      )
+    )
+    return { status: 'error' }
+  }
+
```

Comment:
You adding alot of these with just different i18n keys. Maybe make it a function and reuse it. Would make the code more adjustable and readable

Reply:
Yes, might be but i18next has some issues with encapsulated or keys that are under some kind of check.
Like: `varTrue ? 'key.one' : 'key.two'`
Once we run the command to generate the new keys, it just ignore this and none are created.

Reply:
the same with:
`box.warning.epic.{action'}`
it doesn't created the `actions` keys